src/index.js

```diff
@@ -1,35 +1,37 @@
 import React from "react";
 import { render } from "react-dom";
-import { Server, Response } from "miragejs";
 import Root from "./Root";
-import { startApiServer } from "./ApiServer";
 import "Config/axiosGlobalConfig";
 import "typeface-ibm-plex-sans";
 import "Styles/styles.scss";
 
-if (window.Cypress) {
-  new Server({
-    environment: "test",
-    routes() {
-      let methods = ["get", "put", "patch", "post", "delete"];
-      methods.forEach((method) => {
-        this[method]("/*", async (schema, request) => {
-          let [status, headers, body] = await window.handleFromCypress(request);
-          return new Response(status, headers, body);
+(async () => {
```

Comment:
This removes the json from the bundle. Definitely something we should look at for our other apps to make sure that we aren't including dev files in the final build unncessarily.